<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widgetLeftFunctionArea" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>561</height>
    </rect>
   </property>
   <widget class="QLabel" name="labelHeadPortrait">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>54</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/res/head_0.png</pixmap>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxVoiceManage">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>335</y>
      <width>95</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>通话管理</string>
    </property>
    <widget class="QPushButton" name="pushButtonHookOff_release">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>摘  机</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonHold_retrive">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>保  持</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonConsult">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>求  助</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxLeader">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>450</y>
      <width>95</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>班长席</string>
    </property>
    <widget class="QPushButton" name="pushButtonMonitor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>监  听</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonBreak">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>强  拆</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonInsert">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>强  插</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxStateManage">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>248</y>
      <width>90</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>状态管理</string>
    </property>
    <widget class="QPushButton" name="pushButtonReady_NOT">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>示  忙</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonLogin_out">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>签  入</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxAgentInfo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>90</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>个人信息</string>
    </property>
    <widget class="QLabel" name="labelAgentID">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>AgentID:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditAgentID">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>40</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelAgentName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>65</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>AgentName:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditAgentName">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>83</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelAgentGroupID">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>102</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>AgentGroupID:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditAgentNameGroupID">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>120</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelAgentThisDN">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>AgentThisDN:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditAgentThisDN">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>160</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="widgetChatList" native="true">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>161</width>
     <height>561</height>
    </rect>
   </property>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>161</width>
      <height>561</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>会话列表</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QWidget" name="widgetChatWidget" native="true">
   <property name="geometry">
    <rect>
     <x>278</x>
     <y>9</y>
     <width>491</width>
     <height>561</height>
    </rect>
   </property>
   <widget class="QQuickWidget" name="quickWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>480</width>
      <height>560</height>
     </rect>
    </property>
    <property name="resizeMode">
     <enum>QQuickWidget::SizeRootObjectToView</enum>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header>QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
